src/ipc/ipc4/dai.c

```diff
@@ -36,7 +36,7 @@ void dai_set_link_hda_config(uint16_t *link_config,
 			     struct ipc_config_dai *common_config,
 			     const void *spec_config)
 {
-#if defined(CONFIG_ACE_VERSION_2_0)
+#if ACE_VERSION >= ACE_VERSION_2_0
```

Comment:
FWIW: this is a generic protocol file for all devices, it shouldn't have hardware dependencies.   Rather than update this here for a new platform, maybe it's a good time to get this moved to the platform layer somewhere?